RealmSwift-swift2.0/Realm.swift

```diff
@@ -493,7 +493,13 @@ public final class Realm {
     */
     @warn_unused_result(message="You must hold on to the NotificationToken returned from addNotificationBlock")
     public func addNotificationBlock(block: NotificationBlock) -> NotificationToken {
-        return rlmRealm.addNotificationBlock(rlmNotificationBlockFromNotificationBlock(block))
+        return rlmRealm.addNotificationBlock { rlmNotification, _ in
+            if rlmNotification == RLMRealmDidChangeNotification {
```

Comment:
could combine the if/else paths into just `block(notification: Notification(rawValue: rlmNotification)!, realm: self)`